dist/css/material-wfont.css

```diff
@@ -30890,6 +30896,47 @@ icon-material-blue-grey-A700 {
 .panel-footer {
   background-color: #ececec;
 }
+hr.on-dark {
+  color: #1a1a1a;
+}
+hr.on-light {
+  color: #ffffff;
+}
+@media (-webkit-min-device-pixel-ratio: 0.75), (min-resolution: 120dpi) {
```

Comment:
this prevents compatibility with not webkit/blink browsers. Probably removing the vendor prefix will allow autoprefixer to adds the supported prefixes. Not sure tough, you may try.